maspsx/__init__.py

```diff
@@ -465,6 +465,25 @@ def process_line(self, line):
                     res.append(
                         f"nop # DEBUG: is_addend (r_dest: {r_dest}) '{next_instruction}' does not use $at"
                     )
+            
+            elif not is_addend and r_source == "$2" and r_source == r_dest and int(operand) > 32767:
```

Comment:
Could you do `... (r_source == "$2" or r_source == "$v0") ...`, as I'm trying to support both numbered/named registers.

At some point I'll re-work some of this stuff so we always map registers to, probably, named ones, which will simplify the logic here and elsewhere.

Reply:
I can probably just edit this file within github, let me see.